systemtest/src/main/java/io/strimzi/systemtest/logs/LogCollector.java

```diff
@@ -116,6 +118,13 @@ public LogCollector(ExtensionContext extensionContext, CollectorElement collecto
         }
     }
 
+    public LogCollector() throws IOException {
```

Comment:
should this be above the original initialization? (for a better readability)

Reply:
Yeah I can move it there :)